src/frontend/src/common/components/library/data_subview/components/createTreeModal/index.tsx

```diff
@@ -40,33 +43,35 @@ interface Props {
   failedSamples: any[];
   open: boolean;
   onClose: () => void;
-  setCreateTreeFailed: (hasFailed: boolean) => void;
+  handleCreateTreeFailed: () => void;
+  handleSetCreateTreeStarted: () => void;
 }
 
 export const CreateTreeModal = ({
   sampleIds,
   failedSamples,
   open,
   onClose,
-  setCreateTreeFailed,
+  handleCreateTreeFailed,
+  handleSetCreateTreeStarted,
 }: Props): JSX.Element => {
   const [treeName, setTreeName] = useState<string>("");
   const [isTreeNameTooLong, setTreeNameTooLong] = useState<boolean>(false);
   const [isTreeBuildDisabled, setTreeBuildDisabled] = useState<boolean>(false);
-  const [treeType, setTreeType] = useState<string>("");
-  const [isContextual, setContextual] = useState<boolean>(true);
-  const [isLocal, setLocal] = useState<boolean>(false);
+  const [treeType, setTreeType] = useState<string>("TARGETED");
+  const [isTargeted, setTargeted] = useState<boolean>(true);
+  const [isNonContextualized, setNoneContextualized] = useState<boolean>(false);
```

Comment:
I think this is a typo: `setNoneContextualized` vs `setNonContextualized` (and then match the `isNonContextualized`). Code will work fine as is, but probably nice to bring it in to alignment.

Reply:
oh no that is a typo! good catch

Reply:
i removed this as feedback to one of your other comments